test/test_samplers.py

```diff
@@ -225,3 +227,23 @@ def simulate_one():
     sample = sampler.sample_until_n_accepted(10, simulate_one)
     assert 10 == len(sample.get_accepted_population())
     sampler.cleanup()
+
+
+def test_redis_catch_error():
+
+    def model(pars):
+        print(f"Model for {pars}")
+        if np.random.uniform() < 0.1:
+            raise ValueError("error")
+        return {'s0': pars['p0'] + 0.2 * np.random.uniform()}
+
+    def distance(s0, s1):
+        return abs(s0['s0'] - s1['s0'])
+    prior = Distribution(p0=RV("uniform", 0, 10))
+    sampler = RedisEvalParallelSampler(port=8775)
```

Comment:
When you have the RedisEvalParallelSampler here, no workers are submitted. Instead, you should use the RedisEvalParallelSamplerServerStarter here.